cmd/main.go

```diff
@@ -207,10 +207,24 @@ func runController(ctx context.Context, opts managerOpts) error {
 		os.Exit(1)
 	}
 
-	td, err := tunneldriver.New(ctrl.Log.WithName("drivers").WithName("tunnel"), tunneldriver.TunnelDriverOpts{
-		ServerAddr: opts.serverAddr,
-		Region:     opts.region,
-	})
+	var comments []tunneldriver.TunnelDriverComments
+
+	if opts.useExperimentalGatewayAPI {
+		comments = append(
+      comments,
```

Comment:
hmm is this checking `gofmt`?

Reply:
fmt here: https://github.com/ngrok/kubernetes-ingress-controller/pull/357/commits/c4ccc465a3e9bc4cedf80809cac95b1ac4b55cf1